effekt/shared/src/main/scala/effekt/core/vm/VM.scala

```diff
@@ -104,7 +104,7 @@ enum VMError extends Throwable {
   case MissingBuiltin(name: String)
   case RuntimeTypeError(msg: String)
   case NonExhaustive(missingCase: Id)
-  case Hole()
+  case Hole(span: effekt.source.Span)
```

Comment:
`Span` currently lives in `effekt.source`, but perhaps it should be somewhere else if we're threading it through.

Reply:
Same for Template if I remember correctly